### ISSUE TITLE
search input box missing

### ISSUE THREAD

**Issue by amirouche** [samuelcolvin/helpmanual.io#5]
I can't find the search box right now.

Please help.

PS: great project idea!

**Comment by samuelcolvin** [comment]
![image](https://user-images.githubusercontent.com/4039449/29040603-9076a934-7ba7-11e7-8396-28faf9c0ec08.png)

Is the page somehow rendering differently for you?

**Comment by amirouche** [comment]
![firefox_screenshot_2017-08-07t18-49-38 251z](https://user-images.githubusercontent.com/7444897/29041059-c0d1e9c2-7bb1-11e7-9d21-24977c448a6d.png)

For some reason ublock origin is hiding the search bar on firefox at least.

**Comment by samuelcolvin** [comment]
I've just tried and it's because `main.js` assumes `window.ga` exists and crashes when it doesn't, and ublock is blocking google analytics.

I'll fix.

Thanks a lot for letting me know about this.

**Comment by samuelcolvin** [comment]
Ok, fixed in c46a9ca.

I did a "dirty" update, eg. I modified the js file under it's old name to avoid having to update all 30k files. So you might need some ctrl+reload wizardry to see the change.

Thanks again for pointing this out, I wonder how many people had been using the site with a blocker like this and hadn't had search.

(then again, I guess those people won't appear in ga at all and won't see ads so they're a dead loss to my ego and my wallet :-()
